src/pages/Vote/Governance/index.tsx

```diff
@@ -107,7 +110,10 @@ const Governance: React.FC = () => {
       total={total}
       limit={limit}
       setCurrentPage={setCurrentPage}
-      createProposal={createProposal}
+      createProposal={payload =>
+        createProposal({ ...payload, accountAddress: account?.address || '' })
```

Comment:
Say a developer refactors this code and calls `createProposal` while the account address is undefined, would the contract return an error or would the proposal still be created with an empty address?
I expect the latter, in which case we don't need to make any modification. Otherwise we'd need to add a safeguard

Reply:
This shouldn't be available because the button is hidden until a user connects a wallet with enough XVS. I can double check the call without an address though